app/widgets/common/deployments/DeploymentActionsMenu.consts.ts

```diff
@@ -0,0 +1,30 @@
+export const actions = Object.freeze({
+    delete: 'delete',
+    forceDelete: 'forceDelete',
+    install: 'install',
+    manageLabels: 'manageLabels',
+    setSite: 'setSite',
+    uninstall: 'uninstall',
+    update: 'update',
+    deployOn: 'deployOn'
+});
+
+export const permissions = {
+    executeWorkflowPermission: 'execution_start',
```

Comment:
To keep consistency, probably better would be to have:
```suggestion
    executeWorkflow: 'execution_start',
```

Reply:
Oh yeah, you're right 😁 
As the adjustment requires also some changes across the file, I've implemented it as a part of 0f95837e69dba5552a737686bf59245235a69bf7, rather than suggestion button 🎉